languages.toml

```
@@ -2336,6 +2338,21 @@ language-servers = [ "jsonnet-language-server" ]
 name = "jsonnet"
 source = { git = "https://github.com/sourcegraph/tree-sitter-jsonnet", rev = "0475a5017ad7dc84845d1d33187f2321abcb261d" }
 
+[[language]]
+name = "ada"
+scope = "source.ada"
+injection-regex = "ada"
+file-types = ["adb", "ads", "gpr"]
+roots = ["alire.toml", "*.gpr"]
+comment-token = "--"
+indent = { tab-width = 3, unit = " " }
```

```suggestion
indent = { tab-width = 3, unit = "   " }
```

Is each indentation level supposed to be three spaces? The unit should be the same number of spaces (or `"\t"` if it's tab indentation)

---

I assumed it from original PR, but according to Ada style guide: https://ada-lang.io/docs/style-guide/Source_Code_Presentation#indentation it should be 3 spaces.

---

Ah ok, if it should be 3 spaces then the `indent` config we should use is

```toml
indent = { tab-width = 3, unit = "   " }
```